### ISSUE TITLE
Documentation not correct for "Advanced Users (Security Profile)"

### ISSUE THREAD

**Issue by jerblack** [yadayada/acd_cli#392]
Since the primary (simple) authentication method stopped working, I was trying to make the ["Advanced Users (Security Profile)" steps](http://acd-cli.readthedocs.io/en/latest/authorization.html) work but the steps talk about creating a security profile, and in the process select options related to the scope and redirect URL, but at no point while creating the security profile are those options presented. It looks like Amazon updated their developer portal since making the Drive API invite only, and now we no longer see the same options.

Can we do this?
1. Figure out how to make this process work.
2. Document it step-by-step here: http://acd-cli.readthedocs.io/en/latest/authorization.html

**Comment by zmwangx** [comment]
It's been a long time since I created my security profile (and IIRC it was indeed quite confusing), but I just tried to create another one, and here are some screenshots. It should answer your question about the redirect URL. As for whitelisting and scope, I don't remember and can't find how to do that, so good luck. Maybe try without that first.

![screen shot 2016-09-01 at 9 33 34 am](https://cloud.githubusercontent.com/assets/4149852/18152545/2e18d9fe-7029-11e6-866c-7c987b2a4881.png)
![screen shot 2016-09-01 at 9 34 04 am](https://cloud.githubusercontent.com/assets/4149852/18152546/2e196446-7029-11e6-92a1-9f7c23193f22.png)
![screen shot 2016-09-01 at 9 34 22 am](https://cloud.githubusercontent.com/assets/4149852/18152547/2e1c8e96-7029-11e6-8ac5-aac038cfff34.png)
![screen shot 2016-09-01 at 9 35 16 am](https://cloud.githubusercontent.com/assets/4149852/18152549/2e2705d8-7029-11e6-9123-5a4fd3004e85.png)
![screen shot 2016-09-01 at 9 35 26 am](https://cloud.githubusercontent.com/assets/4149852/18152548/2e2491f4-7029-11e6-8058-d12796d71562.png)

P.S. I know the app ID is exposed in the screenshots. But don't try to hack me, I'll delete that demo profile anyway 😜

**Comment by zmwangx** [comment]
Turns out the reason I can't figure out how to whitelist the profile is that whitelisting is now invitation-only (thanks @isleshocky77 for [pointing out](https://github.com/yadayada/acd_cli/issues/393#issuecomment-243959597)).

Indeed, https://developer.amazon.com/public/apis/experience/cloud-drive/content/getting-started now reads

> You must be invited into the Amazon Drive API service to whitelist your app for Amazon Drive or you will not be able to send any node-related requests.

and in March it read ([Wayback Machine](https://web.archive.org/web/20160322033228/https://developer.amazon.com/public/apis/experience/cloud-drive/content/getting-started))

> You must whitelist your app for the Amazon Cloud Drive or you are be able to send any node-related requests. You must whitelist your Security Profile by completing the whitelist request form.

**Comment by isleshocky77** [comment]
@zmwangx No problem, was just about to document the reason in this issue as well. I would say you can probably close this issue and one of the other ones.  I was hoping it might be a quick process, but the fact that they had me send an email and not fill out a web form to be invited gives me little hope.

**Comment by zmwangx** [comment]
> I would say you can probably close this issue and one of the other ones.

Well I'm not a collaborator on this repo, just a regular dude trying to help out 😉

**Comment by isleshocky77** [comment]
Change took effect August 2, 2016 per Official Release https://forums.developer.amazon.com/articles/33185/the-new-amazon-drive-api-and-sdks-for-android-and.html

**Comment by jerblack** [comment]
So is that the end of ACD_CLI then?

**Comment by zmwangx** [comment]
> So is that the end of ACD_CLI then?

No, my old profile is working just fine. So should @yadayada's public one. I guess someone just abused the service and the outage is temporary.

**Comment by zmwangx** [comment]
> Change took effect August 2, 2016 per Official Release https://forums.developer.amazon.com/articles/33185/the-new-amazon-drive-api-and-sdks-for-android-and.html

Oh wait, "the new Amazon Drive API"... Does that mean we're on a legacy API that is being phased out? At a glance I don't see any difference in the API documentation though: https://developer.amazon.com/public/apis/experience/cloud-drive/content/nodes.

**Comment by dellipse** [comment]
"The new Amazon Drive API and SDKs for Android and iOS are available by
invitation only, starting August 2nd, 2016"

I didn't think we were using those APIs. We are not running on those
platforms.

**Comment by zmwangx** [comment]
> I didn't think we were using those APIs. We are not running on those platforms.

We *are* using the REST API.

**Comment by isleshocky77** [comment]
@dellipse This was in relation to setting up your own Security Profile. You can't whitelist the app without having the Cloud API enabled. You can no longer have that API enabled without being invited.

**Comment by m4vr0x** [comment]
So if i understand well, new acd_cli users can only use Simple (Appspot) authorization method ? :-(

**Comment by awilliamson** [comment]
@m4vr0x Yet the 'Simple' ( Appspot ) method doesn't work due to #416. This leaves us in a bit of dilemma here. Newly created security profiles cannot be whitelisted, and the old auth method is rate limited to the point of uselessness. @yadayada Any suggestions?
